lib/codebreaker/validation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  module Validation
+    MIN_NAME_LENGTH = 3
+    MAX_NAME_LENGTH = 20
+    def name_is_valid?(name)
+      name.instance_of?(String) && name.length.between?(MIN_NAME_LENGTH, MAX_NAME_LENGTH)
+    end
+
+    def input_is_valid?(input)
+      /^[1-6]{4}$/.match?(input)
```

Comment:
nit: maybe you can do this more flexible?) p.s. it's for situation when we need change game rules and use range `1-3` instead of `1-6`

Reply:
so, do you have suggestions on how to deal with it in regex?
because with constant it doesn't work